<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_DialogDatabaseRunQuery</class>
 <widget class="QDialog" name="Dialog_DialogDatabaseRunQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_OriginalData">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Store the data from your Work Order CSV ?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_Tag1g">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Store the 1Gram Tag data ?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_1gToNg">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Link 1gram Tag to Ngram Tag ?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_ItemToItem">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Link Item to Item hierarchy ?</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_IssueToItem">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update Issue to Item Tag links ?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_TagNg">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Store the NGram Tag data ?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QProgressBar" name="progressBar_DialogDatabaseRunQuery_RunQueries">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="lineEdit_DialogDatabaseRunQuery_Info">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_DialogDatabaseRunQuery_DeleteData">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Remove data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_DialogDatabaseRunQuery_RunQueries">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Store Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
